cookbook/Tip-to-use-statics-files-js-css.md

```diff
@@ -0,0 +1,27 @@
+# Static Files
+
+## How let the Yesod know about your file
+
+Yesod build the static files to use inside the project.
+
+Exemple We have one file in project:
+
+	/static/css/my_css_file.css
+
+When Yesod build the libraries, a variable with name css_my_css_file_css is created to use inside Handlers.
+
+If you don't build, he no know about the file and when you use css_my_css_file_css, it no exists.
+
+	
+So, every new file add in static file you should execute the follows commands
+
+```
+stack clean
+stack build
+```
```

Comment:
I think using something like this is better:
```
stack build --fast --file-watch
```
as that will automatically rebuild the code.

Reply:
@Bernardoow In either of the above command, I don't think `stack clean` is required. Using `stack clean` will make the build slow. Can that be removed ?

Reply:
Hi @psibi good morning !!

When I was learning I changed the name of css/bootstrap.css to css/bootstrap1.css and then I used stack build. Few moments later I changed the name again to css/bootstrap.css and then
used stack build so an error happened saying about missing bootstrap1.css.

What I should put in the cookbook (MAYBE), if you just added file you can skip the stack clean but if you remove or changed name of some file you must run stack clean and then stack build.

What do you think?

Reply:
Once the `stack build --fast --file-watch` is complete, you can re-initiate the build again by just pressing the Return (or Enter) key. That should pick up the change. Does that work for you ?